.github/workflows/build_assets.yml

```diff
@@ -31,8 +31,7 @@ jobs:
           - os: ubuntu-20.04
             TARGET: ubuntu
             CMD_REQS: >
-              pip install -r requirements.txt
-              pip install .
+              pip install -r requirements.txt && pip install .
```

Comment:
@giovanni-guidini FYI, both deps could be passed to the same `pip install` command. There's no technical reason for running it twice here.